source/catalogue/index.md

```diff
@@ -88,3 +88,80 @@ The API provides a comprehensive set of resources to manage the state of your Cl
 [GOV.UK PaaS Technical Documentation for Users](https://docs.cloud.service.gov.uk/)
 
 The  API  handles all requests from the [Cloud Foundry Command Line Interface (CLI)](https://docs.cloudfoundry.org/cf-cli/) which is the primary user interface for users of the platform.
+
+### GOV.UK: Search API
+
+**API Base URL links:**
+
+- [Base URL](https://www.gov.uk/api/search.json)
+
+**API Docs URL links:**
+
+ - [Documentation URL](https://docs.publishing.service.gov.uk/apis/search/search-api.html)
+ 
+**API Description:**
+ 
+GOV.UK Search API allows you to find content on GOV.UK. It's the same API that powers https://gov.uk/search/all and the other dynamic content pages on GOV.UK
+ 
+### GOV.UK: Organisations API 
+
+**API Base URL links:**
+
+- [Base URL](https://www.gov.uk/api/organisations)
+
+**API Docs URL links:**
+
+ - [Documentation URL](https://github.com/alphagov/collections/blob/master/docs/api.md)
+ 
+**API Description:**
+ 
+The organisations API provides information about government organisations
+ 
+### GOV.UK: Governments API 
+
+**API Base URL links:**
+
+- [Base URL](https://www.gov.uk/api/governments)
+
+**API Docs URL links:**
+
+ - [Documentation URL](https://docs.publishing.service.gov.uk/apis/whitehall.html)
+ 
+**API Description:**
+ 
+The governments API lists information about governments back to 1801
+
+### Document Checking System Pilot API
+
+**API Base URL links:**
+
+- [Base URL production](https://<DCS-PRODUCTION-URI>/checks/passport)
+
+- [Base URL testing](https://<DCS-TESTING-URI>/checks/passport)
```

Comment:
List items should be in consecutive lines instead of separated by a blank line (line 139)